doc/dev/cephadm/developing-cephadm.rst

```diff
@@ -124,6 +124,20 @@ This means we should do very few synchronous calls to remote hosts.
 As a guideline, cephadm should do at most ``O(1)`` network calls in CLI handlers.
 Everything else should be done asynchronously in other threads, like ``serve()``.
 
+Note regarding different variables used in the code
+===================================================
+
+* a ``service_type`` is something like mon, mgr, alertmanager etc defined 
```

Comment:
how about using 

```rst
.. describe:: service_type

   something like mon, mgr, alertmanager etc defined in ``ServiceSpec``
```

which is rendered like items like "waiting_for_healthy" in https://docs.ceph.com/en/latest/dev/ .

Reply:
tbh, I don't really care. this is just the developer documentation.